src/DssSpell.sol

```diff
@@ -75,334 +60,214 @@ contract DssSpellAction is DssAction {
-    address internal constant NEW_PIP_MKR                  = 0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b;
-    address internal constant VOTE_DELEGATE_FACTORY        = 0xC3D809E87A2C9da4F6d98fECea9135d834d6F5A0;
-    address internal constant REWARDS_LSMKR_USDS           = 0x92282235a39bE957fF1f37619fD22A9aE5507CB1;
-    address internal constant LOCKSTAKE_MKR                = 0xb4e0e45e142101dC3Ed768bac219fC35EDBED295;
-    address internal constant LOCKSTAKE_ENGINE             = 0x2b16C07D5fD5cC701a0a871eae2aad6DA5fc8f12;
-    address internal constant LOCKSTAKE_CLIP               = 0xA85621D35cAf9Cf5C146D2376Ce553D7B78A6239;
-    address internal constant LOCKSTAKE_CLIP_CALC          = 0xf13cF3b39823CcfaE6C2354dA56416C80768474e;
-
-    // ---------- Wallets ----------
-    address internal constant AAVE_V3_TREASURY   = 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c;
-    address internal constant EARLY_BIRD_REWARDS = 0x14D98650d46BF7679BBD05D4f615A1547C87Bf68;
+    uint256 internal constant FIVE_PT_TWO_PCT_RATE = 1000000001607468111246255079;
+
+    // --- Math ---
+    uint256 internal constant RAD = 10 ** 45;
+
+    address internal immutable USDS                    = DssExecLib.getChangelogAddress("USDS");
+    address internal immutable SUSDS                   = DssExecLib.getChangelogAddress("SUSDS");
+    address internal immutable ILK_REGISTRY            = DssExecLib.getChangelogAddress("ILK_REGISTRY");
+    address internal immutable LITE_PSM                = DssExecLib.getChangelogAddress("MCD_LITE_PSM_USDC_A");
+    address internal immutable MCD_PAUSE_PROXY         = DssExecLib.getChangelogAddress("MCD_PAUSE_PROXY");
+
+
+    // ---------- BASE Token Bridge ----------
+    // Mainnet addresses
+    address internal constant BASE_GOV_RELAY           = 0x1Ee0AE8A993F2f5abDB51EAF4AC2876202b65c3b;
+    address internal constant BASE_ESCROW              = 0x7F311a4D48377030bD810395f4CCfC03bdbe9Ef3;
+    address internal constant BASE_TOKEN_BRIDGE        = 0xA5874756416Fa632257eEA380CAbd2E87cED352A;
+    address internal constant BASE_TOKEN_BRIDGE_IMP    = 0xaeFd31c2e593Dc971f9Cb42cBbD5d4AD7F1970b6;
+    address internal constant MESSENGER                = 0x866E82a600A1414e583f7F13623F1aC5d58b0Afa;
+    // BASE addresses
+    address internal constant BASE_GOV_RELAY_L2        = 0xdD0BCc201C9E47c6F6eE68E4dB05b652Bb6aC255;
+    address internal constant BASE_TOKEN_BRIDGE_L2     = 0xee44cdb68D618d58F75d9fe0818B640BD7B8A7B7;
+    address internal constant BASE_TOKEN_BRIDGE_IMP_L2 = 0x289A37BE5D6CCeF7A8f2b90535B3BB6bD3905f72;
+    address internal constant USDS_L2                  = 0x820C137fa70C8691f0e44Dc420a5e53c168921Dc;
+    address internal constant SUSDS_L2                 = 0x5875eEE11Cf8398102FdAd704C9E96607675467a;
+    address internal constant SPELL_L2                 = 0x6f29C3A29A3F056A71FB0714551C8D3547268D62;
+    address internal constant MESSENGER_L2             = 0x4200000000000000000000000000000000000007;
```

Comment:
nit, Can we prefix `L2` here like other places?

Reply:
I didn't get what you mean

Reply:
Sorry for lacking explanation! In exec sheet and test we use prefix for `L1` or `L2` (eg. `L2TokenBridge`)
I think it'd nicer to use `L2` prefix here as well (eg. `L2_SPELL`) to keep the consistency :)